### ISSUE TITLE
Cannot GET /repository/rpi/

### ISSUE THREAD

**Issue by viktorViktor** [DanielOgorchock/OmniThing#13]
When running
```
sudo apt-get update && sudo apt-get install omnithing
```
I'm getting this:
```
Hit:1 http://archive.raspberrypi.org/debian buster InRelease
Get:2 http://raspbian.raspberrypi.org/raspbian buster InRelease [15.0 kB]
Ign:3 http://omnithing.net/repository/rpi ./ InRelease
Err:4 http://omnithing.net/repository/rpi ./ Release
  404  Not Found [IP: 192.99.169.215 80]
Reading package lists... Done
E: The repository 'http://omnithing.net/repository/rpi ./ Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: Repository 'http://raspbian.raspberrypi.org/raspbian buster InRelease' changed its 'Suite' value from 'testing' to 'stable'
N: This must be accepted explicitly before updates for this repository can be applied. See apt-secure(8) manpage for details.
```

Also visiting http://omnithing.net/repository/rpi gives me
```
Cannot GET /repository/rpi/
```

**Comment by DanielOgorchock** [comment]
Thanks for the report. This should be resolved now as of version 0.7.3.

Note the extra step added to the install instructions:
`curl http://omnithing.net/repository/rpi/KEY.gpg | sudo apt-key add`

Please let me know if this resolves the issue. I don't want to close the issue until it works for someone besides myself.

**Comment by DanielOgorchock** [comment]
I believe this is resolved. Feel free to re-open if not.
